src/components/visualizations/chart/events/setupDrilldownToParentAttribute.ts

```diff
@@ -24,7 +24,8 @@ export function getDDPointsInParentTick(axis: any, tick: IHighchartsParentTick):
     // replace y value by target value for bullet chart target
     ddPoints.forEach(ddPoint => {
         if (get(ddPoint, "series.userOptions.bulletChartMeasureType") === "target") {
-            ddPoint.y = ddPoint.target;
+            const isNullTarget = ddPoint.getClassName().includes("hidden-empty-series");
```

Comment:
I don't think this is acceptable.

Reply:
Should I add another property when we add this class to the ddPoint? I had a doubt

Reply:
![image](https://user-images.githubusercontent.com/31855226/78362162-eb948480-75b9-11ea-88a5-d52ccdcd9bae.png)
when the target is `null` it is hidden by the class anyway, right? Would be possible to keep it `null` ?

Reply:
No we can't, the chart won't be displayed then (probably a limitation due to Highcharts)

Reply:
Then I would prefer adding an extra prop (`isNullTarget`) to the place on the screenshot above ☝️ instead of using the class. I don't want the class to be used for anything else than styling.